greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExport.java

```diff
@@ -0,0 +1,98 @@
+package com.greenback.kit.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AutoExport extends GreenbackObject{
+    private String userId;
+    private String accountingAccountId;
+    private AutoExportRequest parameters;
```

Comment:
Isn't there also an "accountIds" which is a List<String>?

Reply:
the accounts are on the List<Account> from the expand